sim-lib/src/lib.rs

```diff
@@ -182,13 +182,18 @@ pub struct Simulation {
     shutdown_listener: Listener,
     // Total simulation time. The simulation will run forever if undefined.
     total_time: Option<time::Duration>,
+    /// The number of activity results to batch before printing in CSV.
+    batch_print: u32,
 }
 
+const DEFAULT_BATCH_PRINT: u32 = 5;
```

Comment:
Lazy back of envelope for picking a value: 
- We don't want too many events in memory, let's be conservative and say no more than 128kb 
- We don't want to hit disk _too_ often, in case somebody is running on a pi

Each CSV row is ~sorta 130 bytes, which means we can have a buffer of about 1000 events. That seems a little high for "toy" scenarios so let's say handwave handwave 500?

Reply:
> We don't want to hit disk too often, in case somebody is running on a pi

Directly influenced by the frequency of events, so 500 might _feel_ like a large batch size for a user watching the file.
Good thing we have a cli config of the batch size :)